test/integration/testdata/app.yaml.tmpl

```diff
@@ -5,6 +5,13 @@ metadata:
   name: {{.service}}
   labels:
     app: {{.service}}
+{{if eq .perServiceAuth "true"}}
+  annotations:
+    tls.istio.io/80: enable
```

Comment:
I strongly recommend to declare per service TLS through a Istio CRD instead of pushing it through annotations. Istio is building it's own config, grinding istio features down into k8s semantics makes it hard to layer them in future.